Gemfile

```
@@ -11,8 +11,8 @@ group :development do
   # > 2.1.0 causes slowdowns https://github.com/sass/sassc-ruby/issues/189
   gem 'sassc', '2.1.0'
   gem 'sass-embedded', '1.89.2'
-  gem 'rubocop', '1.77.0'
-  gem 'ruby-lsp', '0.24.2'
+  gem 'rubocop', '1.78.0'
```

RuboCop (1.75.5)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `rubocop` should appear before `sass-embedded`.

(Bundler/OrderedGems)